pages/Documents/MessagingChannels/MobileAppMessagingSDKforAndroid/Misc/use-the-liveperson-sdk-android.md

```diff
@@ -385,12 +359,11 @@ The proprietary SDK notification is only for display purposes, interacting with
 
        @Override
         protected void onHandleIntent(Intent intent) {
-            Log.d(TAG, "onHandleIntent: registering the token to pusher");
             String token = FirebaseInstanceId.getInstance().getToken();
             // Register to Liveperson Pusher
             String account = "82055668";
             String appID = "com.shaym.sdk28";
```

Comment:
@lpywu can we change this AppId for a com.liveperson.app or something more generic?